core/server/web/api/v2/admin/uploadHelper.js

```diff
@@ -0,0 +1,27 @@
+const shared = require('../../../shared');
+const config = require('../../../../config');
+
+const multer = {
+    clear: config.get('upload-clear') || true,
+    instance: null
+};
+
+const multerInstance = (multerModule) => {
+    multer.instance = multerModule;
+};
+
+const single = type => (req, res, next) => {
```

Comment:
The `type` name here is a bit confusing because `filename` is usually passed in :)